cypress/integration/pages/details.spec.js

```diff
@@ -127,6 +127,16 @@ describe('Details Page', () => {
       cy.visit('http://localhost:3001/customers/1/view');
       cy.get(vulnerabilities).should('not.exist');
       cy.get(assets).should('not.exist');
-    })
+    });
+
+    it('redirects to the correct url after creating a new snapshot', () => {
+      cy.visit('http://localhost:3001/customers/10/view');
+      cy.get(vulnerabilities).should('exist');
```

Comment:
This is from a prev test

Reply:
It acts weird without this. It clicks the button before the whole box is fully loaded and fails. Line 134 forces it to wait and checks if it's loaded before clicking on _add new snapshot_